campaignion_csv.module

```diff
@@ -114,3 +114,46 @@ function campaignion_csv_campaignion_csv_info() {
   ];
   return $export;
 }
+
+/**
+ * Implements hook_webform_results_download_submission_information_info_alter().
+ *
+ * Add a NID column right before the SID column.
+ */
+function campaignion_csv_webform_results_download_submission_information_info_alter(array &$info, $node = NULL, $options = NULL) {
+  if (!$options || !empty($options['multiple_nodes'])) {
```

Comment:
When is options empty and why do we want a nid column in this case?

Reply:
webform (without patch) doesn’t pass the `$node` and `$options`.